src/ZoneServer/Skills/Handlers/Scout/ObliqueFire.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Threading.Tasks;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Yggdrasil.Extensions;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Scout
+{
+	/// <summary>
+	/// Handles the Scout skill Oblique Fire.
+	/// </summary>
+	[SkillHandler(SkillId.Scout_ObliqueFire)]
+	public class ObliqueFire : ITargetSkillHandler
+	{
+		/// <summary>
+		/// Handles the skill, shoot missile at enemy that spreads to another target.
+		/// </summary>
+		public void Handle(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.TurnTowards(designatedTarget);
+
+			if (designatedTarget == null)
+			{
+				return;
+			}
+
+			if (!caster.Position.InRange2D(designatedTarget.Position, skill.Data.MaxRange))
+			{
+				caster.ServerMessage(Localization.Get("Too far away."));
+				return;
+			}
+
+			this.Attack(skill, caster, designatedTarget);
+		}
+
+		/// <summary>
+		/// Executes the actual attack.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="designatedTarget"></param>
+		private async void Attack(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			var damageDelay = TimeSpan.Zero;
+			var skillHitDelay = TimeSpan.Zero;			
+			var secondHitTime = TimeSpan.FromMilliseconds(100);
+
+			var skillHitResult = SCR_SkillHit(caster, designatedTarget, skill);
+			designatedTarget.TakeDamage(skillHitResult.Damage, caster);
+
+			var skillHit = new SkillHitInfo(caster, designatedTarget, skill, skillHitResult, damageDelay, skillHitDelay);
+
+			designatedTarget.Components.Get<BuffComponent>().Start(BuffId.ObliqueFire_Buff, skill.Level, 0, TimeSpan.FromSeconds(10), designatedTarget);
+
+			Send.ZC_SKILL_FORCE_TARGET(caster, designatedTarget, skill, skillHit);
```

Comment:
> I could`t make it work with Z_KILL_HIT_INFO, in game the client did not respond to the skill altough the mob dies. My solution was Send.ZC_SKILL_FORCE_TARGET(caster, designatedTarget, skill, skillHit);

I understand that this solution works for you, but we should really aim to emulate the game's packets and packet sequences as closely as possible, as to not risk running into any unexpected situations where things break and we don't understand why.

You're saying it didn't work with `ZC_SKILL_HIT_INFO`, but did you try to send `ZC_HIT_INFO` for the second hit?

Reply:
ZC_SKILL_HIT_INFO doesn't work unfortunately, maybe I missed some other packet that I must send before it?
The thing is that ZC_SKILL_FORCE_TARGET fits well and ZC_HIT_INFO does as well.

> You're saying it didn't work with ZC_SKILL_HIT_INFO, but did you try to send ZC_HIT_INFO for the second hit?

I am trying to follow the server packet flow. It does it on the official but with ZC_SKILL_HIT_INFO instead of ZC_SKILL_FORCE_TARGET.

Reply:
Well, we need to figure out the missing part. I'm all for not blindly following the packet logs, but the packets we do send should be the right ones. We don't want to hack our way around such issues, as something is clearly going wrong with our code if the correct packets don't work as intended.